docs/modules/user-guide/partials/proc_devfile-quick-reference.adoc

```diff
@@ -0,0 +1,263 @@
+[id="proc_devfile-quick-reference_{context}"]
+= Devfile quick reference
+
+[role="_abstract"]
+This page quickly introduces you to devfile properties. The code snippets help you get started developing your own devfile projects. The tables show you what the different devfile properties require if you use these properties. Refer to the table of contents to go to the devfile property you want to work with.
```

Comment:
As before, I'm not sure what's an expected language style in this set of docs, maybe a more descriptive and "user welcoming" one is preferred.
On "my" projects writers would avoid describing what users are expected to know (e.g. that code snippets contain code examples or that users can use a table of contents). Users tend to scroll down to code snippets, and the faster they get info, the more efficient our docs are considered.

In the end, our reviewers would cut this intro paragraph to something like:
This page describes devfile properties and requirements for using them.

Reply:
Good suggestion--nice and concise. I've made the change.